app/physician-bio/[slug]/page.tsx

```diff
@@ -1,11 +1,153 @@
+  return physicians.map((evt) => ({ slug: evt.slug }));
+}
+
+function renderRichTextToReactComponent(richText: Document, options = {}) {
+  const defaultOptions = {
+    renderNode: {
+      [BLOCKS.UL_LIST]: (node: any, children: React.ReactNode) => {
+        return <ul className="list-disc">{children}</ul>;
+      },
+      [BLOCKS.OL_LIST]: (node: any, children: React.ReactNode) => {
+        return <ol className="list-decimal">{children}</ol>;
+      },
+      [BLOCKS.LIST_ITEM]: (node: any, children: React.ReactNode) => {
+        return <li className="">{children}</li>;
+      },
+      [BLOCKS.PARAGRAPH]: (node: any, children: React.ReactNode) => (
+        <p>{children}</p>
+      ),
+    },
+  };
+  return documentToReactComponents(richText, { ...defaultOptions, ...options });
 }
 
-export default function PhysicianBio({ params }: { params: { slug: string } }) {
+const ContentParagraph = ({ contentItem }: { contentItem: any }) => {
+  console.log("ContentParagraph - contentItem:", contentItem);
+  return <div>{renderRichTextToReactComponent(contentItem as Document)}</div>;
+};
+
+const Section = ({ section }: { section: BioPageSectionType }) => (
+  <div key={section.sys.id}>
+    <h2 className="text-xl">{section.fields.title}</h2>
+    <div className="text-sm">
+      {section.fields.content.content.map(
+        (
+          contentItem: { content: Array<{ value: string }> },
+          contentIndex: number,
+        ) => (
+          <ContentParagraph key={contentIndex} contentItem={contentItem} />
+        ),
+      )}
+    </div>
+  </div>
+);
+
+export default async function PhysicianBio({
+  params,
+}: {
+  params: { slug: string };
+}) {
+  const docBio = await getPhysicianBioBySlug(params.slug);
+
+  let educationAndCertificatesHeaderRendered = false;
+  let researchInsightsAndPublicationsHeaderRendered = false;
   return (
-    <main>
-      <h1>Physician Bio - {params.slug} </h1>
+    <main className="m-10">
+      <div>
+        <Image
+          src={docBio.physicianPortrait.fields.file.url}
+          alt=""
+          width={docBio.physicianPortrait.fields.file.details.image.width}
+          height={docBio.physicianPortrait.fields.file.details.image.height}
+        />
+        <h1 className="text-lg text-[#800080]">{docBio.physicianName}</h1>
+        <h3 className="text-base">Specializes in:</h3>
+        <div>
+          {renderRichTextToReactComponent(
+            docBio.specialties as unknown as Document,
+          )}
+        </div>
+        <div id="phone-numbers">
+          <p>Appointment Number: {docBio.appointmentNumber}</p>
+          <p>Physician Number: {docBio.physicianNumber}</p>
+        </div>
+      </div>
+      <div className="flex items-center">
+        <hr className="flex-grow border-[#99C221]"></hr>
+      </div>
+      <div id="overview">
+        <h2 className="text-xl text-[#0076AD]">Overview</h2>
+        <p>{renderRichTextToReactComponent(docBio.overview as Document)}</p>
```

Comment:
Remove the paragraph tags from this line, as the rich text component already wraps the content in one